### PR TITLE
Schema fix

### PR DESCRIPTION
**Pull request by hcorg**

### CODE REVIEW

File: sanity-check.sh [hunk 1]
```diff
@@ -14,19 +15,27 @@ printf "\n\nGenerating Makefile files...\n"
 printf "\n\nGenerating and building C files...\n"
 make -C ${OUT_DIR} || { echo "Error building C files" ; exit 1 ; }
 
-printf "\n\nDownloading module schema...\n"
+printf "\n\nDownloading JSON Schema schema..\n"
```
Comment (psockoN7) [inline sanity-check.sh]: One dot is missing :P

File: sanity-check.sh [hunk 2]
```diff
@@ -14,19 +15,27 @@ printf "\n\nGenerating Makefile files...\n"
...
     || { echo "Error getting module json-schema" ; exit 1 ; }
+jsonschema -i ${OUT_DIR}/${MODULE_SCHEMA_FILE} ${OUT_DIR}/schema \
+           || { echo "Invalid JSON schema" ; exit 1 ; }
 
-printf "\n\nDownloading general info schema...\n"
+printf "\n\nDownloading and validating general info schema...\n"
 (cd ${OUT_DIR} && wget --no-cache ${URL}/${GENERAL_SCHEMA_FILE}) \
     || { echo "Error getting general json-schema" ; exit 1 ; }
+jsonschema -i ${OUT_DIR}/${GENERAL_SCHEMA_FILE} ${OUT_DIR}/schema \
+           || { echo "Invalid JSON schema" ; exit 1 ; }
```
Comment (psockoN7) [inline sanity-check.sh:32]: This error message could be more useful if it contained information if the error is from module or general file

**Review by psockoN7 — APPROVED** [review]

File: sanity-check.sh (hunk 2 above)
Comment (hcorg) [reply sanity-check.sh:32]: it will be just below  "Downloading and validating XXXX schema" :)

File: sanity-check.sh (hunk 1 above)
Comment (hcorg) [reply sanity-check.sh]: BAM, git amend and nobody will know

**Pull request merged**